Preprocessing/AudioNormalizer.py

```diff
@@ -0,0 +1,97 @@
+import argparse
+from pathlib import Path
+import sys
+from typing import (
+    List,
+    Optional,
+)
+
+import ffmpeg
+
+from Preprocessing.ErrorHandlingLogger import ErrorHandlingLogger
+from Preprocessing.utils import setup_logger
+
+
+class AudioNormalizer:
+    SUPPORTED_EXTENSIONS: List[str] = [".mp4", ".mkv", ".avi"]
+
+    def __init__(self, input_folder: str, output_folder: str):
+        self.input_folder: Path = Path(input_folder)
+        self.output_folder: Path = Path(output_folder)
+        self.logger: ErrorHandlingLogger = ErrorHandlingLogger(
+            class_name=self.__class__.__name__,
+            logger=setup_logger(self.__class__.__name__),
+        )
+
+    def run(self) -> int:
+        self.logger.info("Starting audio normalization...")
+        try:
+            if not self.input_folder.is_dir():
+                self.logger.error(f"Invalid input folder: {self.input_folder}")
+                return 1
+            self.process_folder()
```

Comment:
```suggestion
            if not self.input_folder.is_dir():
                self.logger.error(f"Invalid input folder: {self.input_folder}")
            else:
                self.process_folder()
```

miało być bez customowych rc spoza loggera, poza tym to wypierdoli error, bo przecież nie sfinalizujesz loggera